### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by samdark**
Mainly to test Codeclimate but it makes code a bit more readable as well.

### CODE REVIEW

**Review by cebe — APPROVED** [review]
For easier review: https://github.com/yiisoft/yii2/pull/13113/files?w=0 (ignoring whitespace changes)

File: framework/base/Component.php [hunk 1]
```diff
@@ -503,19 +507,19 @@ public function off($name, $handler = null)
         if ($handler === null) {
             unset($this->_events[$name]);
             return true;
-        } else {
-            $removed = false;
-            foreach ($this->_events[$name] as $i => $event) {
-                if ($event[0] === $handler) {
-                    unset($this->_events[$name][$i]);
-                    $removed = true;
-                }
```
Comment (SilverFire) [inline framework/base/Component.php]: This change looks wrong. `return` statement is missing in this condition

File: framework/base/Component.php [hunk 2]
```diff
@@ -503,19 +507,19 @@ public function off($name, $handler = null)
         if ($handler === null) {
```
Comment (SilverFire) [inline framework/base/Component.php]: We leave empty line before return statement or not?

File: framework/BaseYii.php [hunk 3]
```diff
@@ -139,20 +139,19 @@ public static function getAlias($alias, $throwException = true)
...
+            foreach (static::$aliases[$root] as $name => $path) {
+                if (strpos($alias . '/', $name . '/') === 0) {
+                    return $path . substr($alias, strlen($name));
                 }
             }
         }
 
         if ($throwException) {
             throw new InvalidParamException("Invalid path alias: $alias");
-        } else {
```
Comment (SilverFire) [inline framework/BaseYii.php]: We leave empty line before return statement or not?

File: framework/BaseYii.php [hunk 4]
```diff
@@ -139,20 +139,19 @@ public static function getAlias($alias, $throwException = true)
...
-            return false;
         }
+        return false;
```
Comment (SilverFire) [inline framework/BaseYii.php]: We leave empty line before return statement or not?

File: framework/base/Controller.php [hunk 5]
```diff
@@ -186,9 +186,8 @@ public function run($route, $params = [])
             return $this->runAction($route, $params);
         } elseif ($pos > 0) {
             return $this->module->runAction($route, $params);
-        } else {
-            return Yii::$app->runAction(ltrim($route, '/'), $params);
         }
+        return Yii::$app->runAction(ltrim($route, '/'), $params);
```
Comment (SilverFire) [inline framework/base/Controller.php]: We leave empty line before return statement or not?

File: framework/base/Event.php [hunk 6]
```diff
@@ -116,20 +116,20 @@ public static function off($class, $name, $handler = null)
...
+            if ($event[0] === $handler) {
+                unset(self::$_events[$name][$class][$i]);
+                $removed = true;
+            }
         }
+        if ($removed) {
+            self::$_events[$name][$class] = array_values(self::$_events[$name][$class]);
+        }
+
+        return $removed;
```
Comment (SilverFire) [inline framework/base/Event.php]: We leave empty line before return statement or not?

**Review by SilverFire — CHANGES_REQUESTED** [review]

File: framework/BaseYii.php (hunk 4 above)
Comment (samdark) [inline framework/BaseYii.php]: We don't have agreement on that as far as I remember. In these cases it depends on if it's a separate logical return block or not.

File: framework/base/Event.php (hunk 6 above)
Comment (dynasource) [inline framework/base/Event.php]: rather not. Don't see the use of it as there is already an empty line because of the bracket

**Review by SilverFire — APPROVED** [review]

File: framework/BaseYii.php (hunk 4 above)
Comment (cebe) [inline framework/BaseYii.php]: we did not do that but at the point where PSR2 was changing from tabs to spaces, these lines were introduced for some reason. In most cases I do not think adding a blank line makes sense.

**Review by dynasource — APPROVED** [review]
the changes look OK. The line breaks are inconsistent though.

**Pull request merged**